### PR TITLE
【Hackathon 6th Fundable Projects 3 No.294】fluid operator random_routing

### PR DESCRIPTION
**Pull request by co63oc**
### PR Category

Others

### PR Types

Others

### Description

迁移 random_routing

### CODE REVIEW

**Comment by co63oc** [comment]
@YuanRisheng CI已完成需要review下

**Comment by co63oc** [comment]
PR-CI-NPU-910B-Paddle 无权限重启
image

**Review by YuanRisheng — DISMISSED** [review]

**Comment by co63oc** [comment]
@zyfncg CI已完成需要review下

**Review by wanghuancoder — DISMISSED** [review]
LGTM

File: paddle/fluid/operators/random_routing_op.cc [hunk 1]
```diff
@@ -1,98 +0,0 @@
...
-  void Make() override {
-    AddInput("Prob", "(Tensor) The input Prob index tensor.");
-    AddInput("TopK_Value", "(Tensor) The input TopK_Value index tensor.");
-    AddInput("TopK_Idx", "(Tensor) The input TopK_Idx index tensor.");
-    AddOutput("Out", "(Tensor) The output random routing tensor.");
-    AddComment(R"DOC(expert_count Operator random routing.)DOC");
-  }
-};
-
-DECLARE_INPLACE_OP_INFERER(RandomRoutingInplaceInferer, {"TopK_Idx", "Out"});
```
Comment (zyfncg) [inline paddle/fluid/operators/random_routing_op.cc:85]: inplace关系需要在yaml中进行配置

Comment (co63oc) [reply paddle/fluid/operators/random_routing_op.cc:85]: 已修改

**Review by zyfncg — APPROVED** [review]

**Pull request merged**